### ISSUE TITLE
Unable to start after upgrade from 0.15 on OSX Mavericks

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#4964]
## Summary

Upgrade scenario from 0.15.1 to 0.16 on OSX Mavericks.

KA Lite 0.15 was set to start on boot (`Auto load the application on login` option was activated in Preferences), and I started the installer without stopping it intentionally. Install process apparently concluded fine, and I saw a notification `KA Lite stopped` at some point during the install, but that was probably only the server, since the app/monitor remained active in the dock and status bar.

However, after the install concluded I opened the app and tried to start the server - nothing apparent happens. See the full logs output below.   

## Branch or installer method

Build 74 - Upgrade scenario from 0.15.1 to 0.16 on OSX Mavericks.

## Traceback or relevant snippet from server.log

After the install finished:
```
2016-03-04 12:27:18 Stopped (1)

2016-03-04 12:27:18 0.16.0

2016-03-04 12:27:18 Stopped (1)

2016-03-04 12:27:41 Stopped (1)

2016-03-04 12:28:41 Stopped (1)

2016-03-04 12:29:52 Stopped (1)

2016-03-04 12:30:50 Stopped (1)

2016-03-04 12:31:51 Stopped (1)

2016-03-04 12:32:42 Stopped (1)

2016-03-04 12:33:51 /bin/bash: /usr/local/bin/kalite: /opt/bamboo_data/xml-data/build-dir/KL-MT16-JOB1/osx/temp/pyrun-2.7/bin/pyrun: bad interpreter: No such file or directory

2016-03-04 12:34:41 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:35:41 Traceback (most recent call last):

2016-03-04 12:35:41   File "<pyrun>/pyrun2.7.py", line 823, in <module>

2016-03-04 12:35:41   File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script

2016-03-04 12:35:41   File "<pyrun>/runpy.py", line 240, in run_path

2016-03-04 12:35:41   File "<pyrun>/runpy.py", line 82, in _run_module_code

2016-03-04 12:35:41   File "<pyrun>/runpy.py", line 72, in _run_code

2016-03-04 12:35:41   File "/usr/local/bin/kalite", line 4, in <module>

2016-03-04 12:35:41     
2016-03-04 12:35:41 __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')

2016-03-04 12:35:41   File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script

2016-03-04 12:35:41   File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script

2016-03-04 12:35:41   File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>

2016-03-04 12:35:41     
2016-03-04 12:35:41 raise RuntimeError("KALITE_PYTHON set to invalid python path")

2016-03-04 12:35:41 RuntimeError
2016-03-04 12:35:41 : 
2016-03-04 12:35:41 KALITE_PYTHON set to invalid python path
2016-03-04 12:35:41 

2016-03-04 12:36:53 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:37:51 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:38:51 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:39:51 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:40:51 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:41:51 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:42:48 Traceback (most recent call last):
```

After I tried to start the server:

```
2016-03-04 12:43:41 Traceback (most recent call last):

2016-03-04 12:43:41   File "<pyrun>/pyrun2.7.py", line 823, in <module>

2016-03-04 12:43:41   File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script

2016-03-04 12:43:41   File "<pyrun>/runpy.py", line 240, in run_path

2016-03-04 12:43:41   File "<pyrun>/runpy.py", line 82, in _run_module_code

2016-03-04 12:43:41   File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:44:41 Traceback (most recent call last):

2016-03-04 12:44:41   File "<pyrun>/pyrun2.7.py", line 823, in <module>

2016-03-04 12:44:41   File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script

2016-03-04 12:44:41   File "<pyrun>/runpy.py", line 240, in run_path

2016-03-04 12:44:41   File "<pyrun>/runpy.py", line 82, in _run_module_code

2016-03-04 12:45:36 Traceback (most recent call last):

2016-03-04 12:45:36   File "<pyrun>/pyrun2.7.py", line 823, in <module>

2016-03-04 12:45:36   File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script

2016-03-04 12:45:36   File "<pyrun>/runpy.py", line 240, in run_path

2016-03-04 12:45:36   File "<pyrun>/runpy.py", line 82, in _run_module_code

2016-03-04 12:45:36   File "<pyrun>/runpy.py", line 72, in _run_code

2016-03-04 12:45:36   File "/usr/local/bin/kalite", line 4, in <module>

2016-03-04 12:45:36     
2016-03-04 12:45:36 __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')

2016-03-04 12:45:36   File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script

2016-03-04 12:45:36   File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script

2016-03-04 12:45:36   File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>

2016-03-04 12:45:36     
2016-03-04 12:45:36 raise RuntimeError("KALITE_PYTHON set to invalid python path")

2016-03-04 12:45:36 RuntimeError
2016-03-04 12:45:36 : 
2016-03-04 12:45:36 KALITE_PYTHON set to invalid python path
2016-03-04 12:45:36 

2016-03-04 12:45:37 Traceback (most recent call last):

2016-03-04 12:45:37   File "<pyrun>/pyrun2.7.py", line 823, in <module>

2016-03-04 12:45:37   File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script

2016-03-04 12:45:37   File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:45:41 Traceback (most recent call last):
  File "<pyrun>/pyrun2.7.py", line 823, in <module>
  File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script
  File "<pyrun>/runpy.py", line 240, in run_path
  File "<pyrun>/runpy.py", line 82, in _run_module_code
  File "<pyrun>/runpy.py", line 72, in _run_code
  File "/usr/local/bin/kalite", line 4, in <module>
    __import__('pkg_resources').run_script('ka-lite-static==0.16.0', u'kalite')
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 729, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/setuptools-15.2-py2.7.egg/pkg_resources/__init__.py", line 1642, in run_script
  File "/Applications/KA-Lite/support/pyrun-2.7/lib/python2.7/site-packages/ka_lite_static-0.16.0-py2.7.egg/EGG-INFO/scripts/kalite", line 115, in <module>
    raise RuntimeError("KALITE_PYTHON set to invalid python path")
RuntimeError: KALITE_PYTHON set to invalid python path

2016-03-04 12:46:41 Traceback (most recent call last):

2016-03-04 12:46:41   File "<pyrun>/pyrun2.7.py", line 823, in <module>

2016-03-04 12:46:41   File "<pyrun>/pyrun2.7.py", line 632, in pyrun_execute_script

2016-03-04 12:46:41   File "<pyrun>/runpy.py", line 240, in run_path

2016-03-04 12:46:41   File "<pyrun>/runpy.py", line 82, in _run_module_code

2016-03-04 12:46:41   File "<pyrun>/runpy.py", line 72, in _run_code

2016-03-04 12:46:41   File "/usr/local/bin/kalite", line 4, in <module>
```
 @aronasorman  @cpauya

**Comment by radinamatic** [comment]
Upon restart, app is nowhere to be seen, neither in Launchpad, nor in `Application > KA Lite folder`, but server can be started/stopped from terminal:

![0 16_mavericks](https://cloud.githubusercontent.com/assets/1457929/13514300/09e45456-e1a8-11e5-81b4-9a6f2aeaae0c.png)

**Comment by radinamatic** [comment]
I couldn't see the sidebar nor topic tree on the video download page in Spanish (which was set as default language on 0.15), so I installed the newest `es-ES` contentpack with `kalite shell`, and I got **two** Spanish languages in the Manage:

![es-0 16_mavericks](https://cloud.githubusercontent.com/assets/1457929/13514582/6ece110c-e1aa-11e5-8438-8ec4674d25c5.png)

**Comment by cpauya** [comment]
Hi @radinamatic - please use [build 83 on Bamboo](http://dungeon.learningequality.org/browse/KL-MT16-93) to verify the fix for this. 

Please note that our latest OS X installer will fail the installation if it finds a loaded KA Lite application.

**Comment by amodia** [comment]
@radinamatic & @cpauya  I can't replicate this issue on build [build 92 on bamboo](http://dungeon.learningequality.org/browse/KL-MT16-92). I tested it on OS X Mavericks.

**Comment by radinamatic** [comment]
This is fixed by [#378](https://github.com/learningequality/installers/pull/378), closing!
